notification/http.go

```diff
@@ -0,0 +1,54 @@
+package notification
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/linksmart/thing-directory/common"
+)
+
+type HTTPAPI struct {
```

Comment:
Calling it SSE is more appropriate (also the file name). SSE uses HTTP internally but a typical HTTP client will not be able to consumer it.